### DISCUSSION TITLE
Can repository owners really see the contributors' IP address ?

### DISCUSSION THREAD

**Discussion by iTrooz** [community/community#28391] [Other Feature Feedback, Questions, & Ideas]
I recently got added to a repository, and the invitation said
```
Owners of XXX will be able to see:

    Your public profile information
    [Certain activity](https://docs.github.com/articles/reviewing-your-security-log/#the-repo-category) within this repository
    Country of request origin
    Your access level for this repository
    Your IP address
```

Is it really true that they can see my *IP address* ? Why is that ? And where do repository owner see this ?

I am personally a bit concerned by this

**Comment by mybearworld** [comment]
bump

**Comment by Sussy-OS** [comment]
Any website can see your ip adress, it's not a huge deal. It just gives general information, like your city. It would actually be more dangerous if somebody just had a raw photo that you took, as that has gps data. TLDR; ip adress is just stuff everybody knows already, photos are more dangerous, remove metadata before sharing (though most message systems do this already.)

**Reply by mybearworld** [reply]
Yes, websites can, but normally not everyone using the website. I trust GitHub with approximately knowing my city, but not every GitHub user.

**Reply by Sussy-OS** [reply]
Yeah, I didn't mean everybody, I meant every website. At least it's not every GitHub user though, only the ones who add you as a contributor.

**Reply by iTrooz** [reply]
> only the ones who add you as a contributor.

Do you know where they can see this info ?

**Reply by mybearworld** [reply]
> Yeah, I didn't mean everybody, I meant every website. At least it's not every GitHub user though, only the ones who add you as a contributor.

I still wouldn't trust everyone adding me as a contributor (or to their organization, I'm pretty sure) with my approximate location.

**Reply by ccolorado** [reply]
> it's not a huge deal

Hackers have been planting malware on their repos as part of job interviews constantly over the last couple of years. 
Extremely highly likely these users know how to exploit the knowledge of your IP.
GitHub isn't clear on when, how or how often my IP gets shared. I think this should be taken more seriously.

**Comment by snoweuph** [comment]
+1 On this. Other Platforms don't give maintainers this Info, why does GitHub?

as there is no official statement of GitHub on this, I will assume that maintainers really can see Contributors IPs

**Reply by snoweuph** [reply]
<https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/displaying-ip-addresses-in-the-audit-log-for-your-organization>

This just lets me assume this as well
